features/step_definitions/is.rb

```diff
@@ -46,3 +46,27 @@
     raise "ImageStreamTag #{istag_name} never created"
   end
 end
+
+Given /^the #{QUOTED} #{QUOTED} CRD was finalized$/ do |crd_type, crd_name|
+  ensure_admin_tagged
+  ensure_destructive_tagged
+  _user = admin
+
+  @result = admin.cli_exec(:get, resource: crd_type, resource_name: crd_name, o: 'yaml')
+  ts1 = @result[:parsed]['metadata']['creationTimestamp']
+  
+  success = wait_for(300) { 
+    @result = admin.cli_exec(:delete, object_type: crd_type, object_name_or_id: crd_name)
+  }
+  unless success
+    raise "The crd #{crd_type} can't be deleted."
+  end
```

Comment:
If you want resource to be gone, you better use step `/^(I|admin) ensures? #{QUOTED} (\w+) is deleted(?: from the#{OPT_QUOTED} project)?( after scenario)?$/`. i.e. `admin ensures "#{crd_name}" #{crd_type} is deleted`

Reply:
My resource is `config.samples.operator.openshift.io`, the shotest name must be `config.samples`.
So I couldn't add it as  an  instance variable under lib/openshift dir. Do you have better idea to use the step here?

Reply:
maybe we can try using `_` in place of dot and see if rest of the logic works correctly

Reply:
Not work...
 `oc delete config_samples_operator_openshift_io cluster --wait=false --config=/home/wxj/workdir/dhcp-140-124-wxj/ocp4_admin.kubeconfig
      
      STDERR:
      error: the server doesn't have a resource type "config_samples_operator_openshift_io" `

Reply:
I thought that you mean you tried to add this as a resource in verification tests. Ok, let me try doing by myself and see how such resources can be handled.

Reply:
Yeah, I tried to add a resources under lib/openshift, like below. Unlucky met the above error.
config_samples_operator.rb
`require 'openshift/cluster_resource'

module BushSlicer
  class ConfigSamplesOperator < ClusterResource
    RESOURCE = 'config_samples_operator_openshift_io'
  end 
end`

Reply:
Please check if #329 does the job for you.

Reply:
`admin ensures "cluster" config_samples_operator_openshift_io is deleted`